docs/starters.yml

```diff
@@ -3793,11 +3793,11 @@
     - Styled Components
     - Sitemap Generation
     - Google Analytics
-- url: https://agility-gatsby-starter.netlify.com
+- url: https://agility-gatsby-starter-gatsbycloud.netlify.com
   repo: https://github.com/agility/agility-gatsby-starter
   description: Get started with Gatsby and Agility CMS using a minimal blog.
   tags:
-    - CMS:Other
+    - CMS:Agility CMS
```

Comment:
```suggestion
    - CMS:Agility
```

Reply:
I agree it does sound redundant, but **Agility CMS** is our name and we need to use this format for SEO. There are other companies named **Agility** (not related to CMS) so it's important that we are consistent in how we use our company name publicly.